resources/views/front-end/photo_album_list.blade.php

```diff
@@ -0,0 +1,85 @@
+@push('styles')
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+@endpush
+@extends('layouts.app')
+
+@section('content')
+    <div class="container intro-container">
+        <div class="card">
+            <div class="card-body">
+                <h2 class="card-title">Foto albums der Esac</h2>
+                Hier zijn fotos van klimweekenden, zomertochtjes en borrels te vinden!
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row d-flex align-items-stretch align-items-center">
+            @foreach ($albumsToFrontEnd as $id=>$album)
+                <div class="col-sm-4 d-flex flex-wrap">
+                    <div class="card w-100 position-relative">
+                        <a href="photoalbums/{!! $album['photoalbum']->id !!}">
+                            <img class="card-img-top" src="{!! $album['thumbnail'] !!}" alt="Card image cap">
+                        </a>
+                        <div class="card-body">
+                            <a href="photoalbums/{!! $album['photoalbum']->id !!}">
+                                <h4 class="card-title">{!! $album['photoalbum']->title !!}</h4>
+                                <p class="card-text text-body">{!! $album['photoalbum']->description !!}</p>
+                            </a>
+                        </div>
+                        <div class="card-footer bg-white p-3">
+                            <a class="btn btn-outline-primary" href="photoalbums/{!! $album['photoalbum']->id !!}">Zie meer</a>
+                        </div>
+                    </div>
+                </div>
+            @endforeach
+        </div>
+
+        <div class="mt-2 mt-auto">
+            <h5>Album toevoegen </h5>
```

Comment:
Zou je hier een translation kunnen gebruiken zodat in de Engelse versie hier ook de correcte titel komt te staan.